app/chitter_app.rb

```diff
@@ -0,0 +1,76 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+require_relative 'helpers'
+
+class ChitterApp < Sinatra::Base
+
+  use Rack::MethodOverride
+  register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+
+  get '/' do
+    @peeps = Peep.all.reverse
+    erb :'landing_page'
+  end
+
+  get '/users/register' do
+    @user = User.new
+    erb :'users/register'
+  end
+
+  get '/sessions/sign_in' do
+    erb :'sessions/sign_in'
+  end
+
+  post '/users/new_user_authenticate' do
```

Comment:
this route name is incorrect, it should just be `post '/users'`.  You are not authenticating the user at this stage.